enhancements/microshift/microshift-router-configuration.md

```diff
@@ -0,0 +1,530 @@
+outside of the cluster.
+Building on [LoadBalancer controller enhancement](loadbalancer-service-support.md),
+the service is exposed using the node IP and configured ports.
+
+All the IPs where the router is exposed need to be part of the `LoadBalancer`
+service status definition. In order to simplify configuration the router will
+allow specifying only IP addresses and interface names.
+
+MicroShift will default to listen to external IPs, the service IP and the
+apiserver IP, as this was the previous behavior and it needs to keep it for
+compatibility. Any advanced configuration, such as multiple interfaces, VLANs,
+etc. will need user's configuration. Any user's configuration will override all
+of the defaults.
+
+If the configuration includes duplicates in the form of same entries in the
+same list or referring to the same IP in different ways (hostnames or NICs),
+these will be ignored without warnings/errors.
+
+The NIC IP addresses will be automatically picked up by MicroShift's
+[service controller](loadbalancer-service-support.md). This is described in
+more details in [this section](#using-loadbalancer-service-type).
+
+### Allow specific IP addresses
+This may be seen as a special case of firewalling, but using IP addresses
+instead of ports. Using the same `NetworkPolicy` resources, IPs may be
+specified to allow traffic from them and reject it from any other source.
+
+Please note this is a manual procedure for the user. Documentation will be
+added on how to perform the required actions.
+
+More information in the design details section.
+
+### Workflow Description
+**cluster admin** is a human user responsible for configuring a MicroShift
+cluster.
+
+1. The cluster admin adds specific configuration for the router prior to
+   MicroShift's start.
+2. After MicroShift started, the system will ingest the configuration and setup
+   everything according to it.
+3. The router will be enabled/disabled, be exposed on the specified ports,
+   listen on the specified IPs, and allow connections from only specified the
+   IP addresses, all according to the cluster admin-provided configuration.
+   This is reflected in the status in the LoadBalancer-type service for the
+   router (if the router is enabled).
+
+### API Extensions
+As described in the proposal, there is an entire new section in the configuration:
+```yaml
+ingress:
+  status: <Managed|Removed> # Defaults to Managed.
+  ports:
+    http: <int> # Defaults to 80.
+    https: <int> # Defaults to 443.
+  expose: # Defaults to IPs in the host. Details below.
+    - <NIC name|IP address>
+    - ...
+```
+
+For more information check each individual section.
+
+### Topology Considerations
+#### Hypershift / Hosted Control Planes
+N/A
+
+#### Standalone Clusters
+N/A
+
+#### Single-node Deployments or MicroShift
+Enhancement is solely intended for MicroShift.
+
+### Implementation Details/Notes/Constraints
+The default router is composed of a bunch of assets part of the MicroShift
```

Comment:
I think you are missing a word or phrase here.  
```suggestion
The default router is composed of a bunch of assets that are embedded as part of the MicroShift
```

Reply:
Done